disnake/audit_logs.py

```diff
@@ -720,6 +721,9 @@ def target(
         Object,
         None,
     ]:
+        if self._target_id is None:
```

Comment:
https://canary.discord.com/channels/808030843078836254/913779868985090089/1063912656303575050

As you mentioned, this unfortunately won't work for some audit log types; would have to revert https://github.com/DisnakeDev/disnake/pull/915/commits/9970b3a164820f812b1abe9a470f3081c40093cb as there doesn't seem to be a better solution for now :/

Reply:
9ada52721f443ae67cfd0ff2e5afef5e40bf267c